src/content/getting-started/cloud-provider-accounts/azure/index.md

```diff
@@ -126,6 +127,17 @@ Please remember to change the `roleDefinitionId` in case you would like to use y
 
 This command should be run for all subscriptions that are used for Giant Swarm workload clusters as well as the management cluster that orchestrates it all.  
 
+## Accept legal terms for deeployment of Flatcar image
+
+Giant Swarm deploys Flatcar image developed by Kinvolk from Azure Marketplace. In order to be able to run the image, it is required by Azure to accept the legal terms.
```

Comment:
should we link somewhere? like to the marketplace or something?

Reply:
yeah just had same thought reading this

Reply:
Done